doc/classes/String.xml

```diff
@@ -920,66 +920,93 @@
 			<return type="bool" />
 			<param index="0" name="right" type="String" />
 			<description>
+				Returns [code]true[/code] if both strings are not equal.
```

Comment:
IMO, this (and the other equal/not equal operators) could define what "equal" actually means for two strings, considering there are languages were you ought not to compare strings directly to check if they match and have the exact same char sequence, instead of checking for identity.

Reply:
So it could be more verbose and say something like "... are the same length and contain the same sequence of characters"?

Reply:
Pushed, could you have a second look?

Reply:
I think it's okay now.